Tella/Data/Crypto/CryptoManager.swift

```diff
@@ -316,14 +323,62 @@ extension CryptoManager: CryptoManagerInterface {
     }
     
     func decrypt(_ data: Data) -> Data? {
-
+        
         guard let metaPrivateKey = self.metaPrivateKey else {
             debugLog("Failed to recover private key", space: .crypto)
             return nil
         }
         return decrypt(data, metaPrivateKey)
     }
     
+    func encryptFilePath(inputFileURL: URL, outputFileURL: URL) -> Bool {
+        do {
+            
+            guard let publicKey = recoverKey(.PUBLIC) else {
+                debugLog("Failed to recover public key", space: .crypto)
+                return false
+            }
+            
+            guard let encryptionKeyData = publicKey.getData() else {
+                debugLog("Failed to recover private key data", space: .crypto)
+                return false
+            }
+
+            let crypt = try Cryptor(inputFileURL: inputFileURL, outputFileURL: outputFileURL, encryptionKeyData: encryptionKeyData, cryptoOperation: .encrypt)
+            try crypt.cryptFile()
+
+            return true
+        }
+        catch let error {
+            debugLog("Error encrypt\(error)", space: .crypto)
+            return false
+        }
+    }
+    
+    func decryptfilePath(inputFileURL: URL, outputFileURL: URL) -> Bool {
+        do {
+            
+            guard let metaPrivateKey = self.metaPrivateKey else {
+                debugLog("Failed to recover private key", space: .crypto)
+                return false
+            }
+            
+            guard let encryptionKeyData = metaPrivateKey.getData() else {
+                debugLog("Failed to recover private key data", space: .crypto)
+                return false
+            }
+
+            let crypt = try Cryptor(inputFileURL: inputFileURL, outputFileURL: outputFileURL, encryptionKeyData: encryptionKeyData, cryptoOperation: .decrypt)
```

Comment:
Let's `Separate Concerns` : Separate the concerns of recovering the private key and decrypting the file?

Reply:
maybe a function to decrypt the file like this : 
`private func decryptFile(inputFileURL: URL, outputFileURL: URL, encryptionKeyData: Data) throws {}`

Reply:
and a function -> is now named `decryptFile(at:outputTo:)` for clarity.


`func decryptFile(at inputFileURL: URL, outputTo outputFileURL: URL) -> Bool {}`